gcc/d/d-lang.cc

```diff
@@ -208,21 +206,23 @@ deps_write (Module *module, OutBuffer *buffer, unsigned colmax = 72)
 	Module *m = depmod->aimports[i];
 
 	/* Ignore compiler-generated modules.  */
-	if (m->ident == Id::entrypoint && m->parent == NULL)
+	if (m->ident == Identifier::idPool ("__entrypoint")
```

Comment:
Would it make sense to use a constant for `"__entrypoint"`?

Reply:
The `Id::xxx` is just a micro optimisation.  I don't really think its worth having our own cache on the C++ side.